### ISSUE TITLE
Feature request: split output file by lines count \ file size

### ISSUE THREAD

**Issue by deathmorlock** [davidmoten/big-sorter#2]
Hello again!
It also will be very nice to add split function for output.
For example1:

```
.output(file_out)
.split(1000,bylines)  
result: file01.txt; file02.txt etc.  with 1000 lines in each.

```
For example2:

```
.output(file_out)
.split(100,byFileSize)  
result: file01.txt; file02.txt etc.  with ~100 mb each   (Lines must be intact )

Good:
test
lines
here

Bad:
test
lines
he
```

**Comment by deathmorlock** [comment]
Hello again!
It also will be very nice to add split function for output.
For example1:
```
.output(file_out)
.split(1000,bylines)  
result: file01.txt; file02.txt etc.  with 1000 lines in each.

```
For example2:
```
.output(file_out)
.split(100,byFileSize)  
result: file01.txt; file02.txt etc.  with ~100 mb each   (Lines must be intact )

Good:
test
lines
here

Bad:
test
lines
he
```

**Comment by davidmoten** [comment]
When you have an output file run this command to split output into 1GB chunks:

```java
Util.splitBySize(output, Serializer.linesUtf8(), 1024 * 1024 * 1024);
```
Same for `Util.splitByCount`.

Overloads give you precise control over file naming and can provide mulitple input files.

Might make it part of builder later.

**Comment by davidmoten** [comment]
Released to Maven Central 0.1.15, closing this issue as complete
